### ISSUE TITLE
Cache_url issue.

### ISSUE THREAD

**Issue by fatimaasif155** [eramba/docker#38]
Getting an error of DSN string.

Exception: The DSN string '"Redis://?server=redis&port=6379&password=&timeout=3"' could not be parsed.
In [/var/www/eramba/app/upgrade/vendor/cakephp/cakephp/src/Core/StaticConfigTrait.php, line 254]

![image](https://user-images.githubusercontent.com/90022122/206899995-60c2a542-52cc-44de-9dbf-a9a296431fa8.png)

**Comment by shrkz1** [comment]
Hello,
can you copy paste `docker inspect eramba` output here?

otherwise its strange behavior, maybe try to modify this line locally and remove the double quotes
https://github.com/eramba/docker/blob/74a5a9b58d3b9cee06c8b15b2670f29edfefae22/.env#L5

so it looks like:
```
CACHE_URL=Redis://?server=redis&port=6379&password=&timeout=3
```

and restart the containers (docker compose down and up again).

**Comment by shrkz1** [comment]
I think we found what is the issue. You probably first installed eramba using this docker repository a while back, then you did updates via web interface (in Settings/Updates) and then cron container stopped working, right? Since you downloaded this repo it was updated and changed a bit. These changes are required to start newer versions of eramba.
You should first stop containers:

```docker compose -f docker-compose.simple-install.yml down```

Then download latest docker repository source files, adjust `.env` file with your existing credentials and configuration and run simple install again to start it up.

**Comment by shrkz1** [comment]
This also required a fix because the problem would occur every update through UX. 
Once this https://github.com/eramba/docker/pull/40 is merged (and released) i suggest everyone to download the latest version of this repository source files and run eramba on that.

**Comment by fatimaasif155** [comment]
Thank you for your response. I tweaked the code a little and it worked.I want to ask a thing. Do we have a way to enable Google SSO with Eramba? I tried but couldn’t find a solution. I’ll be thankful if you respond with a solution. Regards,Fatima AsifOn 12-Dec-2022, at 7:19 PM, Martin ***@***.***> wrote:﻿
This also required a fix because the problem would occur every update through UX. Once this is merged (and released) i suggest everyone to download the latest version of this repository source files and run eramba on that.
